src/Compilers/VisualBasic/Portable/Scanner/ScannerXml.vb

```diff
@@ -786,7 +779,8 @@ CleanUp:
         End Function
 
         Friend Function ScanXmlStringUnQuoted() As SyntaxToken
-            If Not CanGet() Then
+            Dim c As Char
+            If Not TryPeek(c) Then
```

Comment:
It doesn't look like 'c' from this TryPeek call is used. Consider going back to CanGet.